packages/fast-components-react-msft/src/caption/caption.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Caption",
+    "description": "A caption component's schema definition.",
+    "type": "object",
+    "id": "caption",
+    "children": true,
+    "properties": {
+        "tag": {
+            "title": "HTML tag",
+            "type": "string",
+            "enum": [
+                "p",
+                "span",
+                "caption",
+                "figcaption"
+            ],
+            "default": "p"
+        },
+        "level": {
+            "title": "Level",
```

Comment:
This should be `Type level` to align with other usage

Reply:
changed